package.json

```diff
@@ -95,7 +95,7 @@
     "prettier:fix": "nx run-many -t prettier:lint --parallel",
     "prettier:fix:api": "nx run api:prettier:fix",
     "build": "nx run-many -t build -p api web workspace --parallel --maxParallel 3",
-    "build:api": "nx run api:build --configuration=production",
+    "build:api": "nx run api:build --configuration=production && pnpm sentry:sourcemaps",
```

Comment:
@rajdip-b  here i just it here, cause we did not implement sentry everywhere. orelse it would be inside build script.

little confused, about if this is the right place. do let me know about this particular change.

Reply:
I believe this will do just fine, We won't be needing sentry while development (should be optional, i.e, if values are specific, we want the logging to take place).

Reply:
Oh, got it. 👍🏼